recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-graphics/gles-module/gles-module-egl-headers_1.9.bbappend

```diff
@@ -7,3 +7,9 @@ SRC_URI_append = " \
     file://0001-EGL-eglext.h-Include-eglmesaext.h.patch \
 "
 
+do_install_append() {
+    if [ ! -d ${DEPLOY_DIR_IMAGE}/xt-rcar ]; then
```

Comment:
This check is not needed. `install -d` should not overwrite directory.

Reply:
Done